src/Sulu/Bundle/AdminBundle/Resources/js/components/DatePicker/DatePicker.js

```diff
@@ -102,6 +102,14 @@ export default class DatePicker extends React.Component<Props> {
 
     handleInputBlur = () => {
         if (this.inputChanged && typeof this.value === 'string') {
+            const fieldOptions = this.getFieldOptions(this.props.options);
```

Comment:
Maybe we should change `fieldOptions`?
It's more the like component props for the datepicker component, or am I wrong?